virt/virt-2-4-release-notes.adoc

```diff
@@ -87,6 +87,7 @@ from being inadvertently started before an upload has completed.
 //CNV-5272 - Configure scheduling directly from CNV UI.
 
 //CNV-5273 - Attach a config map / secret / service account to a VM from the UI.
+* You can now add xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[secrets, ConfigMaps, and service accounts] to a virtual machine on the *Virtual Machine Overview* -> *Environment* page of the {product-title} web console. You can also remove these resources by using the same form.
```

Comment:
epic nit: 'the same form'?

Reply:
I think I'll change it to "on the same page" to avoid ambiguity of meaning :)